ingestion/src/metadata/cli/docker.py

```diff
@@ -275,10 +274,15 @@ def run_docker(  # pylint: disable=too-many-branches
             logger.info(
                 "Stopping docker compose for OpenMetadata and removing images, networks, volumes..."
             )
-            logger.info("Do you want to Delete the docker mounted volumes from host")
-            user_response = click.prompt("Please enter [y/N]", type=str)
+            logger.info(
+                "Do you want to delete the MySQL docker volume with the OpenMetadata data?"
+            )
+            user_response = input("Please enter [y/N]\n")
             if user_response == "y":
-                shutil.rmtree(MAIN_DIR)
+                try:
+                    shutil.rmtree(MAIN_DIR)
+                except FileNotFoundError:
+                    pass
```

Comment:
```suggestion
                try:
                    shutil.rmtree(MAIN_DIR)
                    logger.info("MySQL docker volume data deleted.")
                except FileNotFoundError:
                    logger.info("Deleting MySQL docker volume data skipped since it was not found.")
```
I would add some info after accepting to delete it.

Reply:
cool, I will add that in my upcoming PR, Thanks